src/ui.form-radio/URadio.vue

```diff
@@ -172,8 +39,8 @@ const emit = defineEmits([
   "update:modelValue",
 ]);
 
-const localValue = ref("");
-const radioName = ref(null);
+const localValue = ref<string | null>("");
```

Comment:
I guess localValue may be anything not only string or null. (relates to all components)